README.md

```diff
@@ -176,7 +200,20 @@ The library is at `build_cmake/libCouchbaseLiteC.so`. (Or `.DLL` or `.dylib`)
 
 ### With CMake on Windows
 
-_(Much like building on Unix. Details TBD)_
+Dependencies:
+* [GCC 7+](http://sourceforge.net/projects/mingw-w64/files/Toolchains%20targetting%20Win32/Personal%20Builds/mingw-builds/installer/mingw-w64-install.exe/download)
+* [CMake 3.9+](https://github.com/Kitware/CMake/releases/download/v3.19.0-rc1/cmake-3.19.0-rc1-win64-x64.msi)
+* [Git 2.xx.x](https://git-scm.com/download/win)
+* [Visual Studio 2017 or 2019](https://visualstudio.microsoft.com/thank-you-downloading-visual-studio/?sku=Community&rel=16)
+
+1. Clone the repo.
+2. Run the powershell script `./jenkins/jenkins_win.ps1`
+3. If build fails.
+    1. Make a directory build_cmake i.e. `mkdir build_cmake`
+    2. Find BuiltInWebSocket.cc file and add `#include <functional>` at line 33
```

Comment:
We should fix compile errors in the code, not list them in the README. If this is still happening with the current master branch (we don't have any open issues about it), please add the fix to the PR or file a separate one.